nifi-nar-bundles/nifi-py4j-bundle/nifi-py4j-bridge/src/main/java/org/apache/nifi/py4j/PythonProcess.java

```diff
@@ -287,6 +309,13 @@ String resolvePythonCommand() throws IOException {
 
 
     private void setupEnvironment() throws IOException {
+        // Environment creation is only necessary if using PIP. Otherwise, the Process requires no outside dependencies, other than those
+        // provided in the package and thus we can simply include those packages in the PYTHON_PATH.
+        if (!isUseVirtualEnv()) {
+            logger.debug("Will not create Python Virtual Environment because PIP is disabled in nifi.properties");
```

Comment:
Very minor, but general usage of `pip` is all lowercase and opposed to all uppercase. More importantly though, the message does not reflect the behavior, as it is only based on the the presence of packaged dependencies.
```suggestion
            logger.debug("Python Virtual Environment not created: Python Processor packaged with dependencies");
```